flow360/component/simulation/models/volume_models.py

```diff
@@ -170,11 +171,10 @@ class ActuatorDisk(Flow360BaseModel):
     Note that `center`, `axis_thrust`, `thickness` can be acquired from `entity` so they are not required anymore.
     """
 
-    name: Optional[str] = pd.Field(None)
-    type: Literal["ActuatorDisk"] = pd.Field("ActuatorDisk", frozen=True)
     entities: Optional[EntityList[Cylinder]] = pd.Field(None, alias="volumes")
-
     force_per_area: ForcePerArea = pd.Field()
+    name: Optional[str] = pd.Field(None)
+    type: Literal["ActuatorDisk"] = pd.Field("ActuatorDisk", frozen=True)
```

Comment:
moved to end of the list for typehints:
<img width="658" alt="Screenshot 2024-07-11 at 20 12 06" src="https://github.com/user-attachments/assets/212a3cc7-0c90-4c80-95f0-4954c01f8e03">